ramp-engine/ramp_engine/base.py

```diff
@@ -3,6 +3,17 @@
 
 logger = logging.getLogger('RAMP-WORKER')
 
+log_file = "worker.log"
```

Comment:
I have 2 questions here:

* we are always logging in the same file whatever and whenever a worker is started?
* If the log start to be too long it will be a problem. Thus the current config take care about splitting the file and renaming it something like `worker.log.1`, ...?

Reply:
Basically something like that: https://stackoverflow.com/questions/39442558/logging-rewrite-if-size-gets-too-big

Reply:
> we are always logging in the same file whatever and whenever a worker is started?

I am not sure about the worker. It looks like it would be quite difficult to to deal with this: https://stackoverflow.com/questions/15096090/python-separate-processes-logging-to-same-file . It depends on how useful the worker logs are, for now it seems the dispatcher log is of most use.